pkgs/development/compilers/dtc/default.nix

```
@@ -14,19 +12,20 @@ stdenv.mkDerivation rec {
     sha256 = "0li992wwd7kgy71bikanqky49y4hq3p3vx35p2hvyxy1k0wfy7i8";
   };
 
+  buildInputs = [ libyaml ];
   nativeBuildInputs = [ flex bison pkg-config which ] ++ lib.optionals pythonSupport [ python swig ];
 
   postPatch = ''
     patchShebangs pylibfdt/
   '';
 
-  makeFlags = [ "PYTHON=python" ];
+  makeFlags = [ "PYTHON=python" "check" ];
   installFlags = [ "INSTALL=install" "PREFIX=$(out)" "SETUP_PREFIX=$(out)" ];
 
```

```suggestion
  makeFlags = [ "PYTHON=python" ];
  installFlags = [ "INSTALL=install" "PREFIX=$(out)" "SETUP_PREFIX=$(out)" ];

  doCheck = true;
```

We have a `checkPhase` built-in so this is a less surprising way to make sure checks are run

---

A thank you. Putting "check" there already felt a bit dirty. xD

Fixed it.